packages/slds-linter/src/index.ts

```diff
@@ -22,7 +22,8 @@ const program = new Command();
 program
   .name('linting-cli')
```

Comment:
FYI - In this [PR](https://github.com/salesforce-ux/stylelint-sds/pull/50/files) Kenam trying to change this linting-cli as we moved away. So this may cause merge conflict based on who goes first.

LGTM. Thanks.